gnocchiclient/shell.py

```diff
@@ -61,6 +61,7 @@ class GnocchiCommandManager(commandmanager.CommandManager):
         "archive-policy-rule list": ap_rule_cli.CliArchivePolicyRuleList,
         "archive-policy-rule show": ap_rule_cli.CliArchivePolicyRuleShow,
         "archive-policy-rule create": ap_rule_cli.CliArchivePolicyRuleCreate,
+        "archive-policy-rule rename": ap_rule_cli.CliArchivePolicyRuleRename,
```

Comment:
is this the only value that we ever anticipate being able to modify? if not, it seems safer to make this a generic update command rather than rename specific.